config/riff-streaming.yaml

```diff
@@ -6589,7 +6589,7 @@ metadata:
 ---
 apiVersion: v1
 data:
-  processorImage: projectriff/streaming-processor-native:1.0.0-SNAPSHOT-20191223112938-672714499ed0c511@sha256:41ad7b6b350370b5e53cc718eb7cde49d7f9f7e22e33a3204ae0a4158f798819
+  processorImage: ericbottard/processor-native@sha256:9e1a16a68bd58e61c34e6bfc0af4c25fbd3f04fa2a045f0cd156c0ed71c8a641
```

Comment:
Or wait for https://github.com/projectriff/streaming-processor/pull/42 to be merged